server/lib/nl/common/rank_utils.py

```diff
@@ -55,6 +55,157 @@
     (types.TimeDeltaType.CHANGE, types.RankingType.LOW):
         False,
 }
+# Set of stat vars that need to check which facet is being used before doing
+# ranking because they have facets that are too fine in granularity.
+# This list of svs are svs in the medium embedding index that have a BLS source.
+SVS_TO_CHECK_FACET = set([
```

Comment:
Maybe move this to a separate file?

Reply:
saw that there was a constants file, moved it there